src/compiler/diagnosticMessages.json

```diff
@@ -603,10 +603,6 @@
         "category": "Error",
         "code": 1191
     },
-    "Module '{0}' has no default export.": {

```

Comment:
The solution isn't to delete this message, it's to specialize it when the original module is declared using `export =`